source/io/module_javaScriptInvoke.js

```diff
@@ -338,6 +324,19 @@
             };
         };
 
+        var createJavaScriptParametersArray = function (isInternalFunction, parametersPointer, parametersCount) {
+            var parameters = new Array(parametersCount);
+            for (var index = 0; index < parametersCount; index += 1) {
+                var parameterValueRef = createValueRefFromPointerArray(parametersPointer, index);
+                if (!isInternalFunction) {
```

Comment:
Can we remove the unnecessary not check? ie both cases are defined so no need for inversion

Reply:
Done.